frontend/src/components/common/form/StepForm/SteppedForm.test.tsx

```diff
@@ -58,7 +59,7 @@ const Component = () => {
 
 describe('SteppedForm', () => {
   it('component renders correctly', () => {
-    const tree = renderer.create(<Component />).toJSON();
-    expect(tree).toMatchSnapshot();
+    const { container } = render(<Component />);
```

Comment:
renderer.create doesn't support modals and causes warnings in all subsequent tests if not cleaned up properly.